package.json

```diff
@@ -66,6 +66,7 @@
     "lint-staged": "^15.2.10",
     "prettier": "^3.3.3",
     "sass": "^1.80.6",
+    "sass-embedded": "^1.81.0",
```

Comment:
Should we consider using this package not only for tests, but also to build the application itself, since it has better performance?
https://nextjs.org/docs/app/building-your-application/styling/sass#implementation